### ISSUE TITLE
On providing the command 'Show numbers' respective numbers are not getting highlighted for Navigation controls

### ISSUE THREAD

**Issue by ayushjai19** [microsoft/WinUI-Gallery#2087]
### Which version of the app?

WinUI 3 Gallery

### Description

Environment Details:
OS Version: Dev (OS Build 27881.1000)
App: WinUI 3 Gallery
App Version: 2.6.0.0 (SDK 1.7)
AT: Voice access

Repro Steps:
Launch the WinUI 3 Gallery app.
Navigate to Navigation controls(Fundamentals, Design, Accessibility etc).
Expand the Controls and provide command as 'Show Numbers' and observe the behavior.
Actual Result:
On providing the command 'Show numbers' respective numbers are not getting highlighted for Navigation controls(Fundamentals, Design, Accessibility etc) when it is in expanded state.

Expected Result:
On providing the command 'Show numbers' respective numbers should get highlighted for Navigation controls(Fundamentals, Design, Accessibility etc) when it is in expanded state.

User Impact:
Users who rely on Voice commands, will face difficulty in accessing the Controls if Numbers are not highlighted.

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

### Windows version

Windows 11 24H2 (26100)

**Comment by niels9001** [comment]
I am not able to repro this at all times. I'm seeing the same behavior as in e.g. the `Photos` app or other built-in apps using a `NavigationView`. It seems to be up to Voice Access to determine if all items should be shown or not, and in some cases it does show all the numbers based on what other windows are opened. So if there's a bug here, it's on `NavigationView` and thus external.
